src/pages/cart-page/cart-control-product/cart-control-product.tsx

```diff
@@ -0,0 +1,80 @@
+import { IconButton, useMediaQuery } from "@mui/material";
+import { Dispatch, FC } from "react";
+import RemoveOutlinedIcon from "@mui/icons-material/RemoveOutlined";
+import AddOutlinedIcon from "@mui/icons-material/AddOutlined";
+import { CartProductModel } from "../../../api/api-cart/api-get-cart";
+import { useCartContext } from "../../../context/use-cart-context";
+import { ControlBox, CountBox, TypographyQuantity } from "../cart-page.styled";
+import { breakpointViewSize } from "../../../state/state";
+
+interface DrawerCartBoxItemControlInterface {
+  setCartItemQuantity: Dispatch<React.SetStateAction<number>>;
+  cartItemQuantity: number;
+  cartItem: CartProductModel;
+}
+
+const DrawerCartBoxItemControl: FC<DrawerCartBoxItemControlInterface> = ({
+  cartItemQuantity,
+  setCartItemQuantity,
+  cartItem,
+}) => {
+  const cartContext = useCartContext();
+
+  const countCartProduct = (action: string) => {
+    cartContext.setIsActive(!cartContext.isActive);
+    if (action != "addition") {
+      return setCartItemQuantity(cartItemQuantity - 1);
+    }
+    return setCartItemQuantity(cartItemQuantity + 1);
+  };
+
+  return (
+    <ControlBox
+      sx={{
+        marginTop: useMediaQuery(breakpointViewSize) ? "null" : "20px",
```

Comment:
Have you tested that? If you use hook in other place than top of the file it won't work properly.

Reply:
Please verify if marginTop can accept null as value? In terms of css i don't think so

Reply:
I don't understand, can you explain to me?